src/vite.config.js

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import commonjs from 'vite-plugin-commonjs'
+
+export default defineConfig({
+	root: './src',
+	build: {
+		outDir: './server/public',
+		commonjsOptions: {
+			// exclude: [/TimeInterval.js$/],
```

Comment:
This `// exclude: [/TimeInterval.js$/],` comment will be removed in a later commit.